tools/clang/test/HLSLFileCheck/hlsl/functions/recursion/patch_constant_func_recursion.hlsl

```diff
@@ -0,0 +1,50 @@
+// RUN: %dxc -E HSmain -T hs_6_0 %s | FileCheck %s
```

Comment:
Given there is code that is expressly for HS shaders compiled into libraries, I think there should be a test for that with a function that is annotated with `shader("hull")` and also `patchconstantfunc("fooey")`